pdr_backend/pdr_dashboard/util/db.py

```diff
@@ -94,24 +88,29 @@ def _init_feeds_data(self):
         )
 
     @enforce_types
-    def _init_feed_payouts_stats(self):
-        return self._query_db(
-            f"""
+    def _init_feed_payouts_stats(self, start_date: Union[UnixTimeMs, None] = None):
```

Comment:
here as well... set self.start_date and just use it inside without passing it around